playbooks/roles/load-balancer-v2/files/backend.map.ctmpl

```diff
@@ -1,11 +1,8 @@
 {{- $prefix := "/ocim/instances" }}
-{{- range nodes }}
-{{- $path := (printf "%v@%v" $prefix .Datacenter) }}
-{{ range $ocim_id, $pairs := tree $path | byKey }}
-{{- $domain := "domain_slug" }}
+{{ range $instance_id, $pairs := tree $prefix | byKey }}
+{{- $domain_slug := (index $pairs "domain_slug").Value }}
 {{- $load_balanced_domains := (index $pairs "domains").Value | parseJSON }}
 {{- range $load_balanced_domain := $load_balanced_domains }}
-{{ $load_balanced_domain }} be-{{ $domain }}
-{{- end }}
+{{ $load_balanced_domain }} be-{{ $domain_slug }}
```

Comment:
I do not speak Go. Feel free to tell me to go read a book or something, but perhaps you'd be kind enough to put a few `{{ /* brief comments */ }}` in the ctmpl files?

The `tree $prefix | byKey` and `(index $pairs "domain_slug").Value` are particularly opaque.

Reply:
Haha, I also don't speak golang templates, had to learn this on the spot yesterday. I'll include comments!

Reply:
Left some comments now - hope that's useful.